docs/index.md

```diff
@@ -55,59 +55,6 @@ Trivy is integrated with many popular platforms and applications. The complete l
 - [VS Code plugin](https://github.com/aquasecurity/trivy-vscode-extension)
 - See [Ecosystem] for more
 
-### General usage
```

Comment:
the heading here says this is the "quick start". this change is making the quickstart incomplete. we should reconcile the 2 quickstarts the we now have.